.github/workflows/get-version.yml

```diff
@@ -178,19 +189,33 @@ jobs:
               ;;
             *)
               # Any branch name
-              GITHUB_BRANCH_BASE_REF_NAME="$(gh pr view $BRANCHNAME -q .baseRefName --json headRefName,baseRefName,state)"
-              GITHUB_BRANCH_PR_STATE="$(gh pr view $BRANCHNAME -q .state --json headRefName,baseRefName,state)"
-              echo "release=`date +%s`.`echo ${{ github.sha }} | cut -c -7`" >> $GITHUB_OUTPUT
-
-              # Check if pull request branch targeting develop
-              if [[ "$GITHUB_BRANCH_BASE_REF_NAME" == "develop" ]] && [[ "$GITHUB_BRANCH_PR_STATE" == "OPEN" ]]; then
+              echo "Github repository context var: $GITHUB_REPOSITORY"
+              if [[ "${{ inputs.pull_request_environment }}" == "internal_contributor" ]]; then
+                echo "Internal contribution."
                 echo "release_cloud=1" >> $GITHUB_OUTPUT
                 echo "release_type=$GITHUB_RELEASE_TYPE" >> $GITHUB_OUTPUT
-              else
-                echo "release_cloud=$GITHUB_RELEASE_CLOUD" >> $GITHUB_OUTPUT
+                # GITHUB_BRANCH_BASE_REF_NAME="$(gh pr view $BRANCHNAME -q .baseRefName --json headRefName,baseRefName,state)"
```

Comment:
@tuntoja , what should we do with these comment ?

Reply:
they could be removed,, case is already handled in the develop branches (or should be)